plugins/modules/ipa_sudorule.py

```diff
@@ -359,6 +381,16 @@ def ensure(module, client):
             if not module.check_mode:
                 client.sudorule_add_allow_command_group(name=name, item=cmdgroup)
 
+        if deny_cmd is not None:
+            changed = category_changed(module, client, 'cmdcategory', ipa_sudorule) or changed
```

Comment:
I'm not sure, but how does `category_changed` with these parameters determines whether there will be an actual change for `deny_cmd`?

Reply:
This may be naive on my part, but I was following the same pattern for `dney_cmd` as was already in place for `cmd`

Reply:
I'm not sure what exactly is the code doing there either. It might be broken there as well. But I don't really know how IPA works so...

Reply:
Yes,  it's been operating correctly on my tests.